.github/ISSUE_TEMPLATE/support.yml

```diff
@@ -0,0 +1,69 @@
+name: Support
```

Comment:
Actually if we're switching to the `yaml` files you can probably delete both of the Markdown versions!

Reply:
I think there should only be the support.yml file now but let me know if I'm wrong!

Reply:
I'm not seeing the other support-request.md removed, but that's ok! We can always do it in a followup PR

Reply:
Ah weird I thought i did delete the file but i'll try merging this and then will delete in a separate PR as needed!